log_extractor/extractor.py

```diff
@@ -270,15 +216,58 @@ def _create_test_dir(self, line):
             os.makedirs(test_dir_name)
         return test_dir_name
 
-    def parse_art_logs(self, team=None):
+    def unpack_relevant_remote_logs(self, dst, source_object):
+        """"
+        Unpacks tar.gz files containing remote logs in case they are
+        inside archive.
+
+        Args:
+            dst (str): Destination path to store the extracted files
+            source_object (str): Object containing log directory information
+
+        Returns:
+
+        """
```

Comment:
It looks fine, but I still want to test it at least on 4.1 and 4.2, also we still have runs for 3.6 and 4.0 that archive and logs names look different

Reply:
We don't run 4.0 anymore, but you 're right about 3.6.. I 'll have to make it working.